### ISSUE TITLE
BUG: Sync Status - Wrong Symbol

### ISSUE THREAD

**Issue by daGscheid** [stereum-dev/ethereum-node#794]
**Describe the bug**
Whenever the Sync Status should reach InSync the wrong icon appears on my system

**To Reproduce**
1. Start Installation: One Click Installation
2. Click on: Lighthouse
3. Select Sync Mode - Checkpoint w. f. e. https://goerli.checkpoint-sync.ethpandaops.io
4. Wait for Installation

**Expected behavior**
When reaching InSync status the icon should reflect that in green, currently it is red 

**Screenshots**
![image](https://user-images.githubusercontent.com/82385103/191589866-c6913d00-bad6-4512-8dbd-8b5f9eb3a46f.png)

**Desktop (please complete the following information):**
 - Windows 11
 - 2.0.0-RC-4 

**Node (please complete the following information):**
 - OS: Ubuntu 22.04
 - Lighthouse 

**Additional context**
I used Checkpoint Sync to sync faster. Works great. You should try it as well

**Comment by daverolo** [comment]
The error icon is expected in your submitted case (screenshot) because GETH has obviously a sync issue (values 0/0).

As stated in #601:

ALL CLIENTS Up to date (= in sync) -> GREEN
AT LEAST ONE client is synchronizing (= not in sync) -> BLUE
ERROR in SYNCH (for any client) -> RED

**Comment by daverolo** [comment]
After further investigation i still found a bug here.

You was probably confused because before your consensus client was in sync you had the blue "synchronizing" icon (on my screenshot below represented by Prysm):

![grafik](https://user-images.githubusercontent.com/107847185/191733387-8c225885-877b-4edf-8fd4-0c4659ea8ef7.png)

and as soon as your consensus client (Lighthouse) was in sync, you got the red error icon:
![grafik](https://user-images.githubusercontent.com/107847185/191733784-e63e8611-7139-4663-a303-be4c8605dc18.png)

So the real bug is that there was/is no error icon while the consensus client is synchronizing although the execution client had/has an sync-error.

To summarize:
On both screenshots above a red error icon is expected because at least one client has a sync-error.

Please confirm @daGscheid.

**Comment by daverolo** [comment]
The be absolutely sure we all understand the same regarding #601 please also confirm the cases below @daGscheid 

### Terms

- Term **in-sync** or **synchronized**: The client is synchronized.
- Term **not-in-sync** or **synchronizing**: The client is synchronizing (and therefore not synchronized)
- Term **sync-error:** The client can for **any** reason not synchronize at all (e.g.: service down, unreachable, stuck).

### Cases

1. All clients are in-sync (which implies they are currently synchronized without sync-error)
AAA 999/999
BBB 999/999
CCC 999/999
Expected icon -> GREEN

3. All clients not-in-sync (which implies they are currently synchronizing without sync-error)
AAA 465/999
BBB 372/999
CCC 774/999
Expected icon -> BLUE

4. At least one client is in-sync / At least one client is not-in-sync / No client has a sync-error
AAA 999/999
BBB 278/999
CCC 999/999
Expected icon -> BLUE

5. At least one client is in-sync / At least one client is not-in-sync / At least one client has a sync-error
AAA 999/999
BBB 278/999
CCC 0/0
Expected icon -> RED

**Comment by daGscheid** [comment]
I confirm both.

Short update to my problem: 

The Execution Client has a really weird behaviour on this. 
Try out a checkpoint sync and you will see that, Geth Sync only "seems to start" as soon as there is a full synced consensus client. 

Funny enough if you leave it running, is just works fine after a while. 
F.e.:
![image](https://user-images.githubusercontent.com/82385103/191769872-f15ca8ba-53f3-4dd4-9a3e-6677b7c8375e.png)

**Comment by daverolo** [comment]
> The Execution Client has a really weird behaviour on this.

You're right, it will be even more "weird" when using BESU instead of GETH. I had a voice discussion about that because of #785 with @stefa2k and he can explain that better to you.

Anyway, i guess we agree that the behaviour of the ECs is not related to this bug. The implementation is currently wrong [as mentioned in my previous comment](https://github.com/stereum-dev/ethereum-node/issues/794#issuecomment-1254905563) and needs to be fixed.

**Comment by daGscheid** [comment]
// Left Open until changes are discussed. Have to change logic if clients follow logic consensus > execution.

**Comment by daverolo** [comment]
> // Left Open until changes are discussed. Have to change logic if clients follow logic consensus before execution for sync

Changes discussed on September 27 via voice meeting.
It seems only GETH is waiting for the CC to go in-sync before the client starts to synchronize.
[This GIST ](https://gist.github.com/daverolo/a70100e18c17a3f58cb90d743a5cca00) should cover all possible scenarios for the reworked widget (which takes into account #601, #785, #794 and #799).
